src/Functions/throwIf.cpp

```diff
@@ -22,6 +22,11 @@ namespace ErrorCodes
 namespace
 {
 
+/// The regex-based code style check script in CI complains when it sees "ErrorCodes:: ErrorCode" (space added to avoid another match).
+/// Because this expression is only used in this file, don't add some suppression mechanism to the already complex style checker, instead
+/// work around by creating a namespace alias.
+namespace ErrorCodeAlias = ErrorCodes;
+
```

Comment:
I was wondering what da heck style check is complaining about :)

Reply:
It says that "ErrorCodes::E" is used but not defined :grimacing: ... It's basically a shortcoming of being based on regexes.